File: app/src/main/java/habibkazemi/ir/lingualyrics_android/util/NetworkUtil.java
package habibkazemi.ir.lingualyrics_android.util;

import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;

/**
 * Created by habibkazemi on 5/21/18.
 */

public class NetworkUtil {
    public static boolean isOnline(Context context) {
        ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
        NetworkInfo netInfo = connectivityManager.getActiveNetworkInfo();
        return (netInfo != null && netInfo.isConnected());
    }
}

File: app/src/main/java/ir/habibkazemi/lingualyrics/util/NetworkUtil.kt
package ir.habibkazemi.lingualyrics.util

import android.content.Context
import android.net.ConnectivityManager

/**
 * Created by habibkazemi on 5/21/18.
 */

object NetworkUtil {
    fun isOnline(context: Context): Boolean {
        val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        val netInfo = connectivityManager.activeNetworkInfo
        return netInfo != null && netInfo.isConnected
    }
}
